src/planscape/eval/compute_conditions.py

```diff
@@ -219,4 +222,4 @@ def average_weighted_scores(config: PillarConfig, condition_reader: ConditionRea
     for (condition, _) in conditions:
         if condition is None:
             return None
-    return weighted_average_condition(cast(list[tuple[Condition, float]], conditions))
+    return weighted_average_condition(np.nan, cast(list[tuple[Condition, float]], conditions))
```

Comment:
This is also a change---is it correct?

Reply:
Correct, I should probably leave a TODO to make this more clear